Feature/lan

JeromSar opened the pull request:
Co-author: Casper Henkes

Overall changes
* Added Policy and Guidelines related to LAN committee
* Added notion of Wyvern funds (managed by Chief of Finance)
* Improved cross-referencing
* Improved wording
* Minor changes

cashenkes commented on appendix/guidelines.tex:
```diff
@@ -1,134 +1,237 @@
-    \item At the end of each Wyvern term, a meeting must be held to discuss the continuation of the Wyvern project. In this meeting, new members ('potential members') may be present to join the Wyvern server.
+    \item At the end of each Wyvern term, a meeting must be held to discuss the continuation of the Wyvern community. In this meeting, new members ('potential members') may be present to join the Wyvern server.
+
+    \item If during a meeting, a potential member agrees to join the Wyvern community, he or she becomes a Wyvern member and gains voting rights and financial responsibility with respect to the contribution of the Wyvern server for the coming term.
```
for "this" term

cashenkes commented on appendix/guidelines.tex:
```diff
@@ -1,134 +1,237 @@
+    \item Members may join the Wyvern community with approval of the Wyvern community through a vote.
+
+    \begin{item}
+        All members must pay a contribution fee to support the upkeep of the server.
```
funds are not only for the server

cashenkes commented on section/administration.tex:
```diff
@@ -1,12 +1,13 @@
 \section{Administration}
 \label{sec:administration}
-To ensure that the server is operated properly, careful thought must be put towards the administration of a server. In particular, it is important a user is appointed to maintain financial responsibility to ensure payments are received and bills are paid on time. Another important aspect is technical responsibility. Intuitively, providing each user with root-level access is unwanted as this can e ...
+To ensure that the server is operated properly, careful thought must be put towards the administration of a server. In particular, it is important one user is appointed to maintain financial responsibility in order to ensure each member's contribution is received and bills are paid on time. Another important aspect is technical responsibility. Intuitively, providing each user with root-level acce ...
```
'a server' can be left out

JeromSar commented:
The guidelines and regulations are not situated in a separate document called 'Articles of Association". I've revised, redacted and improved several articles.

Main changes:
* Added establishment date and location
* Revised objective of admin team to objective of Wyvern
* Added conditions for dissolving Wyvern
* Revised (implicit) definition of a member(ship).
  * Membership is now explicitly granted by vote
  * Membership may now be revoked by a unanimous vote
* Revised financial technicalities
  * All financial aspects are now located in a single section
  * Removed introductory paragraph, this is background information (and is found in the meeting notes)
  * All members now have explicit obligation to pay (or leave per section 1)
* Revised definitions of community, administration, and committee meetings.

cashenkes commented on Association/section/democracy.tex:
```diff
@@ -0,0 +1,71 @@
+    \end{item}
+
+    \begin{item}
+        In a successful vote, the most backed option is affirmed. In case of a tie, the administration team may (re)discuss the options, after which the Director settles the vote by either:
```
so in a case that a unanimous VOTE is unsuccesfull since there is a tie, can the administration team then choose which option they want?

JeromSar replied:
@cashenkes I see the issue, the "in case" clause in this article predicates a successful vote. The rules of unanimous votes dictate that the vote must be unanimous to be successful.

cashenkes replied:
you could read it as a if succesfull ... else if tie ... statement which could be a problem

JeromSar commented on Association/section/democracy.tex:
```diff
@@ -0,0 +1,71 @@
+    
+    \item Administration and commission meetings are managed internally by the administration team.
+    
+    \item Commission meetings are managed internally by the respective commission.
```
@cashenkes I don't understand. Commission meetings (as described here) are managed by the commission itself.

JeromSar replied:
Ah I see. This comment is supposed to be for the item on line 6. Will correct this.

cashenkes replied:
it seems to be double since item one says that they are both managed by the administration team, and then item 2 says that commission meetings are managed by the commission. So following both items the meetings would be managed by both
